bedwars-plugin/src/main/java/com/andrei1058/bedwars/listeners/BreakPlace.java

```diff
@@ -213,8 +213,8 @@ public void onBlockDrop(ItemSpawnEvent event) {
         //WHEAT_SEEDS
         IArena arena = Arena.getArenaByIdentifier(event.getEntity().getWorld().getName());
         if (arena == null) return;
-        String material = event.getEntity().getItemStack().getType().toString();
-        if (material.equals("SEEDS") || material.equals("WHEAT_SEEDS")) {
+        Material material = event.getEntity().getItemStack().getType();
+        if (material.equals(Material.BED) || material.equals(Material.SEEDS)) {
```

Comment:
This would not work on 1.13 and up. Beds in newer versions are for example RED_BED, WHITE_BED, etc